tests/test-sessions/test-sessions-common/src/test/java/org/eclipse/jetty/server/session/SessionListenerTest.java

```diff
@@ -301,6 +370,22 @@ public void sessionDestroyed(HttpSessionEvent se)
         {
         }
     }
+    
+    public static class ThrowingSessionListener implements HttpSessionListener
+    {
+
+        @Override
+        public void sessionCreated(HttpSessionEvent se)
+        {
+        }
+
+        @Override
+        public void sessionDestroyed(HttpSessionEvent se)
+        {
+            throw new IllegalStateException("Exception during sessionDestroyed");
```

Comment:
We should test a `Throwable` as well.
perhaps something like ... `javax.xml.parsers.FactoryConfigurationError` ?